### ISSUE TITLE
Discontinue the support for LEDA<6

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#1307]
The following files check `CGAL_LEDA_VERSION` to have a different behavior when the version is <5, <6, or >=6. I think we should now always assume version 6 or later.

```
Arrangement_on_surface_2/include/CGAL/Arr_spherical_gaussian_map_3/Arr_spherical_gaussian_map_3.h
Convex_hull_d/include/CGAL/Regular_complex_d.h
Kernel_23/include/CGAL/kernel_to_kernel.h
LEDA/include/CGAL/LEDA_basic.h
Nef_2/include/CGAL/IO/Nef_polyhedron_2_PS_stream.h
Nef_2/include/CGAL/Nef_2/gen_point_location.h
Nef_S2/include/CGAL/Nef_S2/leda_sphere_map.h
Number_types/include/CGAL/leda_bigfloat.h
Number_types/include/CGAL/leda_bigfloat_interval.h
Number_types/include/CGAL/leda_real.h
Number_types/include/CGAL/leda_coercion_traits.h
Number_types/include/CGAL/leda_rational.h
Number_types/include/CGAL/leda_integer.h
Polynomial/include/CGAL/Polynomial.h
STL_Extension/include/CGAL/Handle_with_policy.h
```

**Comment by afabri** [comment]
I agree. What do you expect me to do?  The clean-up?

**Comment by lrineau** [comment]
@afabri Right. I am pretty sure I will not have time to start doing it before my leave.

If you can find time to do it during August, that would be mergable in CGAL-4.9.

**Comment by lrineau** [comment]
@afabri Now that I am back from summer leave, should I handle this issue and remove the support for LEDA 4 and 5?

**Comment by afabri** [comment]
please go ahead

**Comment by lrineau** [comment]
Fix by #1366 in `master` for CGAL-4.9.
